packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -80,8 +86,39 @@ ins::after {
   content: attr(data-a11y-end);
 }
 
-mark { background: cv('warning', 'lightest'); }
+mark {
+  background: cv('warning', 'lightest');
+}
+
+s {
+  text-decoration: line-through;
+}
+
+small {
+  font-size: ms(-1);
+}
+
+kbd {
```

Comment:
do we wanna use `$mono-font-family` here?

Reply:
![image](https://user-images.githubusercontent.com/62259644/78615485-81cce280-7826-11ea-8d4c-e32fb5db42ae.png)

Was going to use it here, but decided against. Our default text flows much better inline with other copy.